src/components/feature-toggles/initialFeatures.ts

```diff
@@ -0,0 +1,10 @@
+export enum FeatureName {
```

Comment:
I think we can delete this file since we are using config/features.ts

Reply:
yes, it's the first approach, good catch!